docs/production/security-model.md

```diff
@@ -1,207 +1,57 @@
-# Security model
-
-This section attempts to document the Zulip security model. It likely
-does not cover every issue; if there are details you're curious about,
-please feel free to ask questions in [#production
-help](https://chat.zulip.org/#narrow/channel/31-production-help) on the
-[Zulip community server](https://zulip.com/development-community/) (or if you
-think you've found a security bug, please report it to
-security@zulip.com so we can do a responsible security
-announcement).
-
-## Secure your Zulip server like your email server
-
-- It's reasonable to think about security for a Zulip server like you
-  do security for a team email server -- only trusted individuals
-  within an organization should have shell access to the server.
-
-  In particular, anyone with root access to a Zulip application server
-  or Zulip database server, or with access to the `zulip` user on a
-  Zulip application server, has complete control over the Zulip
-  installation and all of its data (so they can read messages, modify
-  history, etc.). It would be difficult or impossible to avoid this,
-  because the server needs access to the data to support features
-  expected of a group chat system like the ability to search the
-  entire message history, and thus someone with control over the
-  server has access to that data as well.
-
-## Encryption and authentication
-
-- Traffic between clients (web, desktop and mobile) and the Zulip
-  server is encrypted using HTTPS. By default, all Zulip services
-  talk to each other either via a localhost connection or using an
-  encrypted SSL connection.
-
-- Zulip requires CSRF tokens in all interactions with the web API to
-  prevent CSRF attacks.
-
-- The preferred way to log in to Zulip is using a single sign-on (SSO)
-  solution like Google authentication, LDAP, or similar, but Zulip
-  also supports password authentication. See [the authentication
-  methods documentation](authentication-methods.md) for
-  details on Zulip's available authentication methods.
-
-### Passwords
-
-Zulip stores user passwords using the standard Argon2 and PBKDF2
-algorithms. Argon2 is used for all new and changed passwords as of
-Zulip Server 1.6.0, but legacy PBKDF2 passwords that were last changed
-before the 1.6.0 upgrade are still supported.
-
-When the user is choosing a password, Zulip checks the password's
-strength using the popular [zxcvbn][zxcvbn] library. Weak passwords
-are rejected, and strong passwords encouraged. The minimum password
-strength allowed is controlled by two settings in
-`/etc/zulip/settings.py`:
+# Securing your Zulip server
```

Comment:
The filename of this should change, because it's no longer the security model, it's security instructions.

Reply:
Added a comment to the PR description that we should do this follow-up.